Makefile

```diff
@@ -27,13 +27,27 @@ DUCKDB_BUILD_TYPE=
 ifeq ($(DUCKDB_BUILD), Debug)
 	DUCKDB_BUILD_CXX_FLAGS = -g -O0 -D_GLIBCXX_ASSERTIONS
 	DUCKDB_BUILD_TYPE = debug
+	DUCKDB_MAKE_TARGET = debug
+else ifeq ($(DUCKDB_BUILD), Static)
+	DUCKDB_BUILD_CXX_FLAGS =
+	DUCKDB_BUILD_TYPE = release
+	DUCKDB_MAKE_TARGET = bundle-library
```

Comment:
`static` vs `dynamic` and `debug` vs `release` are orthogonal concepts.
Can we have `ReleaseStatic` instead? (and we'll add `DebugStatic` if/when need be.
(I don't think it's worth it to rename `Release`/`Debug` into `ReleaseDynamic`/`DebugDynamic`)

Reply:
I totally agree, but DuckDB its make targets don't. There's no way to build a "debug" static library version currently, because the bundle-library hardcodes the release path. I'll rename it to ReleaseStatic though.